installer.sh

```diff
@@ -522,3 +556,31 @@ if [[ "$TARBALL" -eq "1" ]] ; then
     ./make_agent_bundle_tarball.sh $TAR_BUNDLE_FLAGS
 fi
 
+# don't start the emulator until after keylime is installed
+if [[ "$TPM_SOCKET" -eq "1" ]] ; then
+    echo
+    echo "=================================================================================="
+    echo $'\t\t\t\tStart TPM emulator'
+    echo "=================================================================================="
+    # starts emulator and IMA stub at boot
+    cd $KEYLIME_DIR/ima_stub_service
+    ./installer.sh
+	if [[ -n "$(command -v service)" ]] ; then
+		service tpm_emulator restart
+	fi
+	if [[ "$TPM_VERSION" -eq "2" ]] ; then
+		echo 'export TPM2TOOLS_TCTI="mssim:port=2321"' >> $HOME/.bashrc && source $HOME/.bashrc
```

Comment:
It looks like there are a lot of mixed tabs and spaces for indentation -- can we go with all spaces?

Reply:
k, i've fixed the settings in my editor to do 4 spaces.  i also replaced all the tabs